README-MALAY.md

```diff
@@ -0,0 +1,142 @@
+- ["How to get started with Open Source by" Sayan Chowdhury](https://www.hackerearth.com:443/getstarted-opensource/) - Artikel ini merangkumi sumber untuk menyumbang kepada sumber terbuka untuk pemula berdasarkan minat bahasa kegemaran mereka.
+- ["Browse good first issues to start contributing to open source"](https://github.blog/2020-01-22-browse-good-first-issues-to-start-contributing-to-open-source/) - GitHub kini membantu anda mencari masalah pertama yang baik untuk mula menyumbang kepada sumber terbuka.
+- ["How to Contribute to Open Source Project" by Maryna Z](https://rubygarage.org/blog/how-contribute-to-open-source-projects) - Artikel secara komprehensif ini ditujukan ke arah perniagaan (tetapi masih berguna untuk penyumbang individu) di mana ia membincangkan mengapa, bagaimana, dan projek sumber terbuka apa yang harus disumbangkan.
+- ["start-here-guidelines" by Andrei](https://github.com/zero-to-mastery/start-here-guidelines) -
+Mari bermula di dunia Sumber Terbuka, bermula di taman permainan Sumber Terbuka. Direka khas untuk tujuan pendidikan dan pengalaman praktikal.
+
+## Enjin carian GitHub secara langsung
+Cari pautan yang mengarahkan secara langsung kepada masalah yang sesuai untuk disumbangkan dalam GitHub.
+- [is:issue is:open label:beginner](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Abeginner)
+- [is:issue is:open label:easy](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aeasy)
+- [is:issue is:open label:first-timers-only](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Afirst-timers-only)
+- [is:issue is:open label:good-first-bug](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Agood-first-bug)
+- [is:issue is:open label:"good first issue"](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A"good+first+issue")
+- [is:issue is:open label:starter](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Astarter)
+- [is:issue is:open label:up-for-grabs](https://github.com/search?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Aup-for-grabs)
+
+## Ekosistem penyumbang Mozilla
+- [Good First Bugs](https://bugzil.la/sw:%22[good%20first%20bug]%22&limit=0) - pepijat yang telah dikenal pasti oleh pemaju sebagai pengenalan yang baik untuk projek ini.
+- [Mentored Bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=mentor%3A%40) - pepijat yang mempunyai mentor yang ditugaskan yang akan berada di IRC untuk membantu anda apabila anda rasa tidak produktif semasa menyelesaikan masalah.
+- [Bugs Ahoy](https://www.joshmatthews.net/bugsahoy/) - laman web yang dikhaskan untuk mencari pepijat di Bugzilla.
+- [Firefox DevTools](http://firefox-dev.tools/) - laman web yang dikhaskan untuk bug yang difailkan untuk alat pembangun dalam penyemak imbas Firefox.
```

Comment:
- add `(developer tools)` next to word `alat pembangun`
- add `(browser)` next to word `penyemak imbas`